app/assets/javascripts/snowcrash/plugins/jquery.treenav.js.coffee

```diff
@@ -29,9 +29,6 @@ do ($ = jQuery, window, document) ->
       if !target.siblings('.children').first().has('li.node').length
         $menu = target.siblings('.children')
         $.get(target.attr 'href')
-        .done ->
-          $menu.find('li.loading').hide()
-          $menu.find('li.error').hide()
```

Comment:
If you're wondering why I removed this, it's because when they're present, the 'Loading' text gets hidden before the tree has even started expanding, so you never see it (for nodes below the top level at least):

![hideloading](https://cloud.githubusercontent.com/assets/2656485/16518571/dc1d77e8-3f83-11e6-8999-30f399058f87.gif)

With the lines removed you can see the "loading" text and the slide looks less jarring:

![withouthideloading](https://cloud.githubusercontent.com/assets/2656485/16518585/ec0a348e-3f83-11e6-8b10-1784b3c0c8b2.gif)